src/ray/common/constants.h

```diff
@@ -23,4 +23,6 @@ constexpr char kTaskTablePrefix[] = "TaskTable";
 
 constexpr char kWorkerDynamicOptionPlaceholderPrefix[] = "RAY_WORKER_OPTION_";
 
+constexpr char kWorkerNumWorkersPlaceholder[] = "RAY_WORKER_NUM_WORKERS";
```

Comment:
Can this just be `RAY_WORKER_OPTION_0`?

Reply:
It can't. This placeholder is meant to be language independent. The idea is to keep `WorkerPool` have no idea how the real argument is, but only need to replace the placeholder with a number.

On the other hand, the full java argument is `-Dray.raylet.config.num_workers_per_process_java=RAY_WORKER_PLACEHOLDER_NUM_WORKERS`. If you replace `RAY_WORKER_PLACEHOLDER_NUM_WORKERS` with `RAY_WORKER_PLACEHOLDER_OPTION_0`, then we have to specificly add the `-Dray.raylet.config.num_workers_per_process_java=` part in C++.

Plus, this argument has nothing to do with dynamic worker options. It's odd to occupy 1 token of dynamic worker options. And we need to make a convention that it would always occupy the first token.